vpr/src/route/route_tree_timing.cpp

```diff
@@ -347,6 +363,7 @@ add_subtree_to_route_tree(t_heap* hptr, t_rt_node** sink_rt_node_ptr) {
 
     //Expand (recursively) each of the main-branch nodes adding any
     //non-configurably connected nodes
+    //Sink is not included, so no need to pass in the node's ipin value.
```

Comment:
I don't see any pin index passed in, so I don't understand the comment.

Reply:
My apologies, I sort of put this in as a reminder for myself that there is no need to pass in any net pin info to this function. But I agree it's confusing. I am taking it out.